src/main/java/io/candydoc/plugin/save_ubiquitous_language/file/UbiquitousLanguageAsFile.java

```diff
@@ -57,4 +64,22 @@ public static UbiquitousLanguageAsFile forFormat(String outputFormat, String out
     }
     throw new IllegalArgumentException("Supported file formats are 'json', 'yml'.");
   }
+
+  public class Manifest {
+    private final List<String> jsonPaths;
```

Comment:
As you're in the `AsFile` adapter, it's not necessary a json output. Moreover, you're not dealing with json paths but with file paths.

Reply:
:+1: